boards/rp-pico/examples/pico_usb_serial_interrupt.rs

```diff
@@ -60,9 +83,12 @@ static mut USB_SERIAL: Option<SerialPort<hal::usb::UsbBus>> = None;
 /// infinite loop.
 #[entry]
 fn main() -> ! {
```

Comment:
Wouldn't be a comment like "never ending function, except when the pico is reset to usb boot." more appropriate?

Reply:
Good spot, this text is actually inconsistent with the example.